sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/HexSuite.scala

```diff
@@ -37,4 +37,10 @@ class HexSuite extends SparkFunSuite {
     assert(Hex.hex(Long.MinValue).toString === "8000000000000000")
     assert(Hex.hex(Long.MaxValue).toString === "7FFFFFFFFFFFFFFF")
   }
+
+  test("SPARK-48671: hex bytes values") {
+    assert(Hex.hex(Array[Byte]()).toString === "")
```

Comment:
assert(Hex.hex(Array[Byte](null)).toString) === ??? 

is this a valid test case?

Reply:
Yes. Both before and after `SPARK-48644` improvement, the result is "".